dbt-cta/facebook_marketing/models/2_partner_matviews/_2_partner_matviews__models.yml

```diff
@@ -0,0 +1,806 @@
+version: 2
+
+models:
+
+  - name: campaign_report
+    description: ''
+    columns:
+      - name: _campaign_report_hashid
+        description: ''
+      - name: date_start
+        description: ''
+      - name: account_id
+        description: ''
+      - name: account_name
+        description: ''
+      - name: campaign_id
+        description: ''
+      - name: campaign_name
+        description: ''
+      - name: clicks
+        description: ''
+      - name: impressions
+        description: ''
+      - name: spend
+        description: ''
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+    tests:
+      - dbt_utils.equal_rowcount:
```

Comment:
oh fascinating!! this is my first time noticing this test - I wonder what it would mean if this one failed, though? It's basically just confirming that the matview refreshed and (at least in terms of rowcount) matches the table created by the `*_base` model? (Are there any other `compare_model` tests that might make sense?)

Reply:
Ahh I mean I agree with your logic here, this seems a fine starting point! Can always revise in the future if we get Older and Wiser 👍